### ISSUE TITLE
Share query functionality.

### ISSUE THREAD

**Issue by vera-liu** [airbnb/caravel#1052]
We want an option in sql lab to get a link of current query, send it to another user. Then the other user can directly open up the link to see the query result

**Comment by vera-liu** [comment]
Waiting on: https://github.com/airbnb/caravel/pull/1073

**Comment by vera-liu** [comment]
https://github.com/airbnb/caravel/pull/1154

**Comment by bkyryliuk** [comment]
@vera-liu - what do you think about specifying database name and no user id in the share link ?

**Comment by vera-liu** [comment]
Opened another issue for this : https://github.com/airbnb/caravel/issues/1310

**Comment by wwj718** [comment]
if i want to share query  , anonymous user  can  access it what should i do  （as a test）
